src/iconsManifest/languages.ts

```diff
@@ -260,6 +260,7 @@ export const languages: ILanguageCollection = {
   purescript: { ids: 'purescript', defaultExtension: 'purs' },
   pyret: { ids: 'pyret', defaultExtension: 'arr' },
   python: { ids: 'python', defaultExtension: 'py' },
+  pythowo: {ids: 'pythowo', defaultExtension: 'pyowo'},
```

Comment:
Is there any extension in the marketplace supporting this language id or did you just add it here?